cluster/plan.go

```diff
@@ -633,6 +649,7 @@ func (c *Cluster) BuildKubeProxyProcess(host *hosts.Host, serviceOptions v3.Kube
 		"hostname-override": host.HostnameOverride,
 		"kubeconfig":        pki.GetConfigPath(pki.KubeProxyCertName),
 	}
+	CommandArrayArgs := map[string][]string{}
```

Comment:
Can we do something similar here as with my previous suggestions to save on unnecessary allocations?